### DISCUSSION TITLE
Grouping data and presets

### DISCUSSION THREAD

**Discussion by echo1377** [akvorado/akvorado#469] [Q&A]
Hello!

Is it possible to add data grouping or sorting in the console?
For example, select destination port 443 and show the amount of traffic on this port against the background of total traffic.

![netflow](https://user-images.githubusercontent.com/81549259/218005139-68227398-f2ae-4e05-9ca9-824684879c99.jpg)

Is it also possible to add presets to the console to quickly change Time range, Dimensions, Limit, Filter?

**Comment by vincentbernat** [comment]
For presets, currently, bookmark the page. I am not sure to have understood your other request. On your screenshot, everything is already destination port 443.

**Reply by echo1377** [reply]
This is just an example of what a sortable field looks like.
The idea is to quickly filter the parameters and plus see the amount of traffic on 443 and the amount of total traffic.
If you use a simple filter "DstPort = 443", then it is difficult to compare the amount of this traffic (443) from the total.

For example, color traffic is DstPort 443 and Other is total traffic

![netflow2](https://user-images.githubusercontent.com/81549259/218043948-f21796d6-3dc8-4e6d-850b-18686d498672.jpg)

**Reply by echo1377** [reply]
I have bookmarked the page but the timespan is not updating automatically. CTRL+F5 doesn't help. Only pressing the Refresh key in the console helps. Time range preset - "Last hour"
Is it possible to fix this? 
![netflow3](https://user-images.githubusercontent.com/81549259/219281149-aa06ba7d-f6d9-494d-9996-65401ba69c69.jpg)

**Reply by vincentbernat** [reply]
That's on purpose to make caching work when you share an URL with someone else. We could maybe be smarter when the difference between the cached date and the current date is high.

**Reply by echo1377** [reply]
Understood. Maybe pre-checking is a really good idea.

**Comment by SamLowrie** [comment]
I stumbled over a similar question: Is it possible to display data in percent against an aggregated value?

For example: How many percent is the traffic from the Netfelix-AS against the rest of the incoming traffic? (Which results in one graph in percent)?

**Reply by vincentbernat** [reply]
This is currently not possible.

**Reply by vincentbernat** [reply]
Building the request to get that information is easy. The difficult task is to have an UI for that. "Total" does not mean much without a filter because it would count flows in both directions for example. It could be an additional filter for the grouping or the ability to use an expression as a dimension.
